### DISCUSSION TITLE
Velocyto.py generates lower counts than Cell Ranger. Can we still use the spliced/unspliced information from velocyto for scVelo analysis?

### DISCUSSION THREAD

**Discussion by hyjforesight** [theislab/scvelo#813] [Q&A]
...
Hello scVelo,
It is recommended to use velocyto.py to generate the spliced and unspliced information from BAM file for the downstream scVelo analysis. However, the velocyto.py generates much lower counts than 10X Cell Ranger did (picture below). It means that the velocyto.py generated loom file doesn't reflect the reality of the experiment. Can we still use the spliced/unspliced information from velocyto.py for scVelo analysis?

![image](https://user-images.githubusercontent.com/75048821/150916464-7eb7766d-d519-4adc-ad27-7a4a0b84592d.png)

Thanks!
Best,
YJ

**Comment by WeilerP** [comment]
@hyjforesight, sorry, I might be missing something: As far as I can see, the only difference between the two Figures is the scale of the x-axis. Isn't this effect removed once the data has been normalized?
Also, which counts are shown on the x-axis for the Velocyto output? Is it the sum of spliced, unspliced and ambiguous? Or is it, e.g., only spliced?

**Comment by JBreunig** [comment]
I believe this is due to the lack of multimapping in both the CellRanger/Velocyto or historically in the Kallisto/Bustools pathways (I think they have added multimapping now [https://github.com/pachterlab/kb_python/issues/127#issue-892197431]).  Genes with high homology (e.g. Sox genes in neural cells) often get dropped in Velocyto matrix generation.  Our solution was to switch to STARsolo to align fastqs and generate velocity matrices.
